locale/ca/get-involved/index.md

```diff
@@ -40,6 +41,6 @@ contribuir de la manera que puguin. Si vols [reportar un error](https://github.c
 - [Grup d'usuaris de Japó](http://nodejs.jp/)
 - [Comunitat Coreana de Node.js](http://nodejs.github.io/nodejs-ko/)
 - [Comunitat de Nicaragua de Node.js](http://nodenica.com/)
-- [Grup de Facebook de llengua castellana per a Node.js](https://www.facebook.com/groups/node.es/)
-- [Comunitat de llenguatge hispà](http://nodehispano.com)
+- [Grup de Facebook en espanyol per Node.js](https://www.facebook.com/groups/node.es/)
```

Comment:
`Grup de Facebook en espanyol de Node.js` I find the `de` better than `per`here.